src/plugins/intel_gpu/tests/fusions/deconvolution_fusion_test.cpp

```diff
@@ -236,80 +243,80 @@ TEST_P(deconv_actv, basic) {
 }
 
 INSTANTIATE_TEST_SUITE_P(fusings_gpu, deconv_actv, ::testing::ValuesIn(std::vector<deconv_test_params>{
-    deconv_test_params{ CASE_DECONV_FP32_1, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP32_2, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP32_3, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP32_4, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP32_5, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP32_6, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP32_7, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP32_8, 2, 3 },
-
-    deconv_test_params{ CASE_DECONV_FP16_1, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP16_2, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP16_3, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP16_4, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP16_5, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP16_6, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP16_7, 2, 3 },
-    deconv_test_params{ CASE_DECONV_FP16_8, 2, 3 },
-
-    deconv_test_params{ CASE_DECONV_U8S8_1, 2, 3 },
-    deconv_test_params{ CASE_DECONV_U8S8_2, 2, 3 },
-    deconv_test_params{ CASE_DECONV_U8S8_3, 2, 3 },
+    deconv_test_params{ CASE_DECONV_FP32_1, 2, 2, 3 },
```

Comment:
What about:
1) Change activation function to onednn-friendly one
2) Add new test case for onednn-unfriendly post operation

Reply:
Done.